complete/completeprocessing.php

```diff
@@ -1,146 +1,172 @@
 <?php
-	$title = 'Boiler Books';
-	$completeactive = "active";
-	include '../menu.php';
+    $title = 'Boiler Books';
+    $completeactive = "active";
+    include '../menu.php';
 ?>
 
-<?php
-include '../dbinfo.php';
-echo '<div class="container">';
-// define variables and set to empty values
-$cost = $comments = $receipt = $purchasedate = "";
-$committee = $_SESSION['committeec'];
-$item = $_SESSION['itemc'];
-$purchaseID = $_SESSION['purchaseIDc'];
-if ($purchaseID == '') {
-	echo 'no purchase';
-	header("Location: index.php");
-}
-else {
-	$sqler = $uploaderr = '';
-	$comments = test_input($_POST["comments"]);
-	$cost = test_input($_POST["cost"]);
-	$purchasedate = test_input($_POST["purchasedate"]);
-	$purchasedateorig = $purchasedate;
-
-	$purchasedate = str_replace('-','/',$purchasedate);
-	$purchasedate = date('Y-m-d H:i:s', strtotime($purchasedate));  
-	$purchasedatetemp = $purchasedate;
-	$receipt = $servername . "/";
-	$target_dir = "../receipts/";
-	$target_dir_save = "/receipts/";
-	$uploadOk = 1;
```

Comment:
STOP USING 1's and 0's FOR TRUE/FALSE

Php has 1) weird dynamic type casting 2) has real true/false types

So don't use 1 and 0